app/stylesheet/application-webpack.scss

```diff
@@ -5,5 +5,5 @@
 @import '~@pf3/timeline/style.css';
 @import '~@manageiq/react-ui-components/dist/quadicon.css';
 @import '~@manageiq/react-ui-components/dist/textual_summary.css';
-@import '~@manageiq/react-ui-components/dist/wooden-tree.css';
 @import '~@manageiq/ui-components/dist/css/ui-components.css';
+@import './wooden-tree.scss';
```

Comment:
Can't we just move it to the tree styling CSS directly? Or is it have to be served through webpack?

Reply:
Yeah this was more of a proof of concept, fixing in https://github.com/brumik/react-wooden-tree/pull/50

Reply:
But when we move the component here, I think it can be with the rest of tree css, yes.